getting-started.md

```diff
@@ -0,0 +1,31 @@
+# Getting started
+
+## Quick Start
+
+Here is a quick start for Zeno players.
+
+**First steps**: [Installation](installation) | [Overview](overview.md)
+
+**Tutorial for beginners**:
+
+ - [Part 1: "Hello world!" in Zeno](#)
```

Comment:
I thought this is already done in Examples..

Reply:
Yep.  I think they fit more under "tutorial", since they're not working examples with a specific theme.